### PR TITLE
Added Pulumi packages

### PR DESCRIPTION
**Pull request by toriancrane**
## Description

This PR adds additional packages to the csharp section to ensure users have them installed.

### CODE REVIEW

**Review by interurban — APPROVED** [review]
looks good

File: themes/default/content/docs/using-pulumi/testing/unit.md [hunk 1]
```diff
@@ -223,14 +223,16 @@ We use the built-in `go test` command, so no need to install anything.
 
 We use [NUnit](https://nunit.org/) test framework to define and run the tests, [Moq](https://github.com/moq/moq4) for mocks, and [FluentAssertions](https://github.com/fluentassertions/fluentassertions) for assertions.
 
-Install the corresponding NuGet packages to your program:
+Install the corresponding NuGet and Pulumi packages to your program:
```
Comment (desteves) [inline themes/default/content/docs/using-pulumi/testing/unit.md:226]: These are all considered NuGet packages. ( remove 'and Pulumi' )

Comment (cnunciato) [reply themes/default/content/docs/using-pulumi/testing/unit.md:226]:
```suggestion
Install the corresponding NuGet packages to your program:
```

**Review by cnunciato — APPROVED** [review]
Approved, but agree with @desteves's comment that "and Pulumi" could be removed.

**Pull request merged**